cfn-resources/global-cluster-config/mongodb-atlas-globalclusterconfig.json

```diff
@@ -0,0 +1,122 @@
+        }
+      },
+      "additionalProperties": false
+    },
+
+    "apiKeyDefinition": {
+      "type": "object",
+      "properties": {
+        "PrivateKey": {
+          "type": "string"
+        },
+        "PublicKey": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "description": "Returns, adds, and removes Global Cluster managed namespaces and custom zone mappings.",
+  "handlers": {
+    "create": {
+      "permissions": [
+
+      ]
+    },
+    "read": {
+      "permissions": [
+
+      ]
+    },
+    "delete": {
+      "permissions": [
+
+      ]
+    }
+  },
+  "primaryIdentifier": ["/properties/ProjectId"],
+  "properties": {
+    "ApiKeys": {
+      "$ref": "#/definitions/apiKeyDefinition"
+    },
+    "ProjectId": {
+      "description": "The unique identifier of the project for the Atlas cluster.",
+      "type": "string"
+    },
+    "ClusterName": {
+      "description": "The name of the Atlas cluster that contains the snapshots you want to retrieve.",
+      "type": "string"
+    },
+    "ManagedNamespaces": {
+      "type": "array",
+      "description": "List that contains comma-separated key value pairs to map zones to geographic regions. These pairs map an ISO 3166-1a2 location code, with an ISO 3166-2 subdivision code when possible, to the human-readable label for the desired custom zone. MongoDB Cloud maps the ISO 3166-1a2 code to the nearest geographical zone by default. Include this parameter to override the default mappings.\n\nThis parameter returns an empty object if no custom zones exist.",
+      "items": {
+        "$ref": "#/definitions/managedNamespace",
+        "type": "object"
+      }
+
+    },
+    "RemoveAllZoneMapping": {
+      "type": "boolean",
+      "description": "Flag that indicates whether all custom zone mapping to be deleted during delete."
+    },
+    "CustomZoneMappings": {
+      "type": "array",
+      "description": "List that contains comma-separated key value pairs to map zones to geographic regions. These pairs map an ISO 3166-1a2 location code, with an ISO 3166-2 subdivision code when possible, to the human-readable label for the desired custom zone. MongoDB Cloud maps the ISO 3166-1a2 code to the nearest geographical zone by default. Include this parameter to override the default mappings.\n\nThis parameter returns an empty object if no custom zones exist.",
+      "items": {
+        "$ref": "#/definitions/zoneMapping",
+        "type": "object"
+      }
+    },
+    "Id": {
+      "type": "string"
+    }
+  },
+  "readOnlyProperties": ["/properties/RemoveAllZoneMapping","/properties/CustomShardKey","/properties/ProjectId","/properties/CustomZoneMappings"],
+  "typeName": "MongoDB::Atlas::GlobalClusterConfig",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git"
+}
```

Comment:
be careful of this EOF i remember i had some issues, the give problems when trying to clone the repo